src/rf/apps/workers/process.py

```diff
@@ -211,6 +212,11 @@ def validate_image(self, record):
         updated = status_updates.mark_image_valid(s3_uuid)
         if updated:
             log.info('Image validated %s', key)
+
+            log.info('Getting metadata from image %s', key)
+            bounds = validator.get_image_bounds()
+            image.set_bounds(bounds)
```

Comment:
I discovered this problem last night before the power went out. We may want to open a separate issue for this.

If an exception is thrown by `validator.get_image_bounds` or `image.set_bounds`, when the queue processor retries this job, it will abort early because "Image ... is already validated". As a result, the handoff job is never queued (even though it has technically been validated).

We probably want to make sure `status_updates.mark_image_valid` is executed *after* the image bounds are saved.